internal/internal_workflow.go

```diff
@@ -149,6 +149,8 @@ type (
 	// Used to truncate internal stack frames from thread stack.
 	unblockFunc func(status string, stackDepth int) (keepBlocked bool)
 
+	ExecutionState int
```

Comment:
```suggestion
	executionState int
```

I think this and all its values should not be exported from this package